### PR TITLE
fix(curriculum): make example less confusing

### LINKED ISSUE #45081: Local Scope and Functions: example needs to be a little less confusing
## Affected page
[Local Scope and Functions
](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/local-scope-and-functions)

## Your code

```




```

## Expected behavior

Once `console.log(loc)` has been used twice inside the caption, in the paragraph below it, `. The console.log(loc) line will throw an error` might be a little confusing, I think we have somehow to mention that we're talking about the second occurrence of it, not the first which is inside the function, just for better quality and clarification.

## Screenshots

Screenshot from 2022-02-11 03-35-29

## System

- Device: [e.g. iPhone6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.4]
- Browser: [e.g. chrome, safari]
- Version: [e.g. 22]

### PR DESCRIPTION
**Pull request by ls-madhu**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #45081

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/local-scope-and-functions.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ myTest();
 console.log(loc);
 ```
 
-The `myTest()` function call will display the string `foo` in the console. The `console.log(loc)` line will throw an error, as `loc` is not defined outside of the function.
+The `myTest()` function call will display the string `foo` in the console. The `console.log(loc)` line (outside the myTest function) will throw an error, as `loc` is not defined outside of the function.
```
Comment (gikf) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/local-scope-and-functions.md:26]:
```suggestion
The `myTest()` function call will display the string `foo` in the console. The `console.log(loc)` line (outside of the `myTest` function) will throw an error, as `loc` is not defined outside of the function.
```
A bit of nitpicking. I wondered also, if simply referring to this `console.log(loc)` as a _second_ one wouldn't work.

---

Note for future - it is recommended to make changes on a separate branch, than the `main` branch. See more about it in point 3 at https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally.

Comment (ls-madhu) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/local-scope-and-functions.md:26]: Sorry, I am new to open source. And thanks for pointing out my mistake.

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me!

**Comment by nhcarrigan** [comment]
Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Review by kmbenjel — COMMENTED** [review]
Still not good enough in the style, already `myTest()` call is outside the function definition.
Could you open a PR again, suggesting:

---
The `myTest()` function call will display the string `foo` in the console. The last line will throw an error, as `loc` is not defined outside of the function.
---

Sufficient and clear ;)

**Comment by gikf** [comment]
That sentence refers to the line with `console.log(log)` specifically. With parenthesized note, for me it is clearer, about which occurrence is that sentence.

**Comment by kmbenjel** [comment]
Agree

**Pull request merged**